src/test/java/org/jabref/logic/integrity/IntegrityCheckTest.java

```diff
@@ -39,197 +37,23 @@
 class IntegrityCheckTest {
```

Comment:
I think, a JavaDoc should be added explaning the difference between this test and the CheckerTests. Here a proposal:

```suggestion
/**
 * This class tests the Integrity Checker as a whole. Aspects are: selected fields, issues arising in a complete BibTeX entry, ... When testing a checker works with a certain input, this test has to go to a test belonging to the respective checker. See PersonNamesCheckerTest for an example test.
 */
class IntegrityCheckTest {
```